### ISSUE TITLE
[design] Supplemental Claims | PACT Act / No Evidence Flow

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#51660]
## Value Statement
**_As a_** Veteran
**_I want to_** test the new Supplemental Claims flow that allows me to submit a claim without evidence if I have a presumptive condition
**_So that_** I can share feedback on the usability of the design and content.

---
## Background Context
- Before we make changes the Supplemental Claim flow in staging, we want to understand how Veterans will interact with it by conducting usability testing. 

## Designs and Build Notes
- [Design WIP here](https://www.sketch.com/s/33fc9b86-6d05-4398-97c6-6dab4af515d7/p/9B9A6622-6A03-4151-99B5-AD62020D958B/canvas)

## Out of scope
- Working code

## Tasks
- [x] Draft initial design
- [x] Share internally
- [x] Stakeholder and SME review
- [x] Design review (office hours)
- [x] Copy review (office hours)
- [x] Legalese review by OAR (email)
- [x] Create prototype
- [ ] [User research](https://github.com/department-of-veterans-affairs/va.gov-team/issues/51573)

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by eileen-coforma** [comment]
Scheduled design and copy review for this week. Need stakeholder review to be scheduled.

**Comment by eileen-coforma** [comment]
Design review feedback:
- Be aware of alert fatigue that's been happening.
- Make use of collapsible alerts where possible.
- User test to see if veterans have confusion about PACT, and if the alert could cause abandonment?

**Comment by saderagsdale** [comment]
Copy review tomorrow afternoon.

**Comment by eileen-coforma** [comment]
- Copy has been reviewed, and updated in designs.
- Designs are ready.
- Sent email to Jane Chi for OAR review of the 'Review and acknowledge this notice of evidence needed'
- Prototype will be done in Figma, and needs configuring.

**Comment by eileen-coforma** [comment]
Waiting on OAR feedback about the Notice of Evidence needed. Have a meeting this Friday with Jane Chi to provide more context.
